### PR TITLE
docs(Elements): update usage to v11 usage guidance

### PR DESCRIPTION
**Pull request by dakahn**
Closes https://github.com/carbon-design-system/carbon/issues/10875

Updates elements usage docs to match our v11 guidance

### CODE REVIEW

**Comment by joshblack** [comment]
@dakahn could you include the type updates in this PR, as well? I think it'd be for this page: https://v11.carbondesignsystem.com/guidelines/typography/code

File: src/pages/guidelines/motion/code.mdx [hunk 1]
```diff
@@ -48,34 +48,61 @@ yarn add @carbon/motion
...
+  transition: opacity motion.motion(standard, productive);
+
+  // Or use a duration
+  transition: opacity motion.$duration-fast-01;
 }
 ```
 
-Both the `motion` function and mixin support passing in the name of the motion
-curve and the mode you want to work in.
+#### API
```
Comment (aagonzales) [inline src/pages/guidelines/motion/code.mdx:93]: Formatting change: we normally add a manual line break between H4s and tables so they're not too close.

image

File: src/pages/guidelines/spacing/code.mdx [hunk 2]
```diff
@@ -52,25 +52,56 @@ yarn add @carbon/layout
...
+@use '@carbon/layout';
+
+.selector {
+  margin-bottom: layout.$spacing-05;
+  width: layout.rem(24px);
+  height: layout.rem(24px);
+}
+```
+
+#### API
```
Comment (aagonzales) [inline src/pages/guidelines/spacing/code.mdx:69]: Add a line break here too

File: src/pages/guidelines/motion/code.mdx [hunk 3]
```diff
@@ -48,34 +48,61 @@ yarn add @carbon/motion
...
-motion modes: productive and expressive. You can access these curves using
-either Sass or JavaScript.
-
 ### Sass
 
-`@carbon/motion` exports a `carbon--motion` function and `carbon--motion` mixin
-that you can use to access the value of a motion curve or include that curve as
-the `transition-timing-function` for a selector. To use these helpers, you can
-do the following in your project:
+The `@carbon/type` package enables you to use typography from the IBM Design
```
Comment (aagonzales) [inline src/pages/guidelines/motion/code.mdx:53]: Is this type content? Is it supposed to be on the motion page?

File: src/pages/guidelines/motion/code.mdx [hunk 4]
```diff
@@ -96,6 +123,21 @@ import { easings, motion } from '@carbon/motion';
 motion('standard', 'productive'); // Returns a string `cubic-bezier()` function
 ```
 
+### Configuration
+
+You can configure parts of the `@carbon/type` package with Sass Modules. For
```
Comment (aagonzales) [inline src/pages/guidelines/motion/code.mdx:128]: Also type content on the motion page?

File: src/pages/guidelines/motion/code.mdx (hunk 1 above)
Comment (dakahn) [reply src/pages/guidelines/motion/code.mdx:70]: let me know if this isn't what you meant

Comment (aagonzales) [reply src/pages/guidelines/motion/code.mdx:70]: Yes, but I think the `<br />` needs to be on its own line. Table is showing up broken now. 
image

**Review by aagonzales — APPROVED** [review]

**Review by joshblack — APPROVED** [review]

**Pull request merged**